knative-kubecon/README.md

```diff
@@ -19,6 +19,8 @@ setup scripts are tailored to facilitate that. To setup a fresh minishift cluste
 we need from Knative installed, run the following scripts:
 
 ```bash
+git clone git@github.com:openshift-cloud-functions/demo.git
+cd demo
```

Comment:
I think this will need to be `cd demo/knative-kubecon` since the scripts folder resides inside there and subsequent commands are also relative to that.

Reply:
Good catch. I also realized the actual repo name was wrong in our clone command and we should clone over https instead of ssh.